src/components/viewer/Upvote.tsx

```diff
@@ -14,19 +14,27 @@ import ppGA from 'pp-ga';
 
 interface IUpvoteProps {
   indirectChildClassName: string;
+
   annotation: AnnotationAPIModel;
+  upvote: AnnotationUpvoteAPIModel;
 
+  fetchUpvote: (url: string) => Promise<object>;
   deleteUpvote: (instance: AnnotationUpvoteAPIModel) => Promise<object>;
   createUpvote: (instance: AnnotationUpvoteAPICreateModel) => Promise<object>;
 }
 
 interface IUpvoteState {
-  initialView: boolean; // used to determine whether edit/delete buttons should be visible
```

Comment:
This field was used by edit/delete buttons to determine whether they are still visible, I suppose that's an unintended change :)

Reply:
(they disappear after some time when the mouse is not over an annotation any more)

Reply:
It was obsolete code after file splitting, this field is used elsewhere :P